addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/webservice.xml

```diff
@@ -4,6 +4,9 @@
 	<bridge-type id="webservice">
 		<label>Tankerkönig Webservice</label>
 		<description>The Tankerkönig Webservice can handle 1 to 10 gas stations.</description>
+        <channels>
```

Comment:
Can you please reformat this is a mixture of tabs and spaces, it should be tabs.